tests/integration/collection_version/updates/add/simple_test.go

```diff
@@ -50,8 +50,6 @@ func TestSchemaUpdatesAddSimpleErrorsAddingSchema(t *testing.T) {
 	testUtils.ExecuteTestCase(t, test)
 }
 
-// This test documents unwanted behaviour, it should fail but it does not, likely due to the `Sources` json
-// deserialization magic.  This bug is documented by https://github.com/sourcenetwork/defradb/issues/3795
```

Comment:
thought: Seems like you this PR should resolve #3795 no?

Reply:
Yes, I've been updating that ticket but forgot to add it to the PR description, thanks.